cpp/data/installer/ub_tools.sql

```diff
@@ -41,20 +41,20 @@ CREATE TABLE delivered_marc_records (
     hash VARCHAR(40) NOT NULL,
     zeder_id VARCHAR(10) NOT NULL,
     delivered_at TIMESTAMP NOT NULL DEFAULT NOW(),
-    journal_name VARCHAR(191) NOT NULL,
-    main_title VARCHAR(191) NOT NULL,
+    journal_name VARCHAR(1000) NOT NULL,
+    main_title VARCHAR(1000) NOT NULL,
     publication_year CHAR(4) DEFAULT NULL,
     volume CHAR(40) DEFAULT NULL,
```

Comment:
not part of this change so far, but: why is CHAR used here instead of VARCHAR? (for volume, issue, pages)